src/components/PoolingWidget/index.tsx

```diff
@@ -192,6 +112,23 @@ const PoolingInterface: React.FC = () => {
   // Avoid displaying an empty list of tokens when the wallet is not connected
   const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
 
+  const methods = useForm({
+    defaultValues: {
+      // @ts-ignore
```

Comment:
eh? You probably need to type `useForm<>`

Reply:
oops